pkg/3scale/amp/operator/apicast_options_provider.go

```diff
@@ -30,6 +30,7 @@ const (
 	APIcastEnvironmentCMAnnotation             = "apps.3scale.net/env-configmap-hash"
 	PodPrioritySystemNodeCritical              = "system-node-critical"
 	CustomPoliciesSecretResverAnnotationPrefix = "apimanager.apps.3scale.net/custompolicy-secret-resource-version-"
+	HPAEnabledWarning                          = "HorzontalPodAutoscaling enabled this overrides values applied to request, limits and replicas"
```

Comment:
```suggestion
	HPAEnabledWarning                          = "HorizontalPodAutoscaling enabled overrides values applied to request, limits and replicas"
```

Reply:
Looks like I am not using this const, will remove it